WordPress/src/main/java/org/wordpress/android/sharedlogin/SharedLoginAnalyticsTracker.kt

```diff
@@ -11,5 +12,16 @@ class SharedLoginAnalyticsTracker @Inject constructor(
 
     fun trackLoginSuccess() = analyticsTracker.track(Stat.SHARED_LOGIN_SUCCESS, emptyMap())
 
-    fun trackLoginFailed() = analyticsTracker.track(Stat.SHARED_LOGIN_FAILED, emptyMap())
+    fun trackLoginFailed(errorType: ErrorType) =
+            analyticsTracker.track(Stat.SHARED_LOGIN_FAILED, mapOf(ERROR_TYPE to errorType))
```

Comment:
I get a null from 
`Tracked: shared_login_failed, Properties: {"error_type":null}`
I guess because we should map to the string

Reply:
Great catch, thanks! I forgot our tracker takes a `Map<String, Any>`. 
I'll fix it and add unit test to this class.